nixos/modules/misc/ids.nix

```diff
@@ -666,6 +667,7 @@ in
       rstudio-server = 324;
       localtimed = 325;
       automatic-timezoned = 326;
+      vtune = 327;
```

Comment:
Do we actually need fixed uids here? We only do this for services that acquire a lot of state and where chown on startup to fixup ids would be too slow.

Reply:
No. I've removed this.